source-reminder/README.md

```diff
@@ -9,60 +9,55 @@ Thus, this tool exists to cross-reference articles on the website and articles s
 identify any pieces of work for which source information has not been submitted.
 
 ## Requirements
+* Docker
 * [Lambda](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=tier%23always-free) (Amazon Web Services)
 * [Google Drive](https://www.google.com/drive/)
-* [Slack SDK](https://slack.dev/node-slack-sdk/)
-* [Gspread](https://github.com/burnash/gspread) 
-* [Client library](https://github.com/googleapis/oauth2client) for OAuth 2.0&mdash;also known as `oauth2client`
-* [Python Requests](https://docs.python-requests.org/)
-* [Python Datetime](https://docs.python.org/3/library/datetime.html)
 
 _We can be loose with the requirements at times, but at least having them pinned to the major version of a package tends to be helpful._
-
-## Google Cloud Integration
-To properly receive the data from an API call, you&rsquo;ll be required at some point to provide what&rsquo;s called a *credentials* file for said account.
-It lives in the Amazon Web Services Lambda computing platform with the rest of the files and responds to webhook requests. Seven libraries are required, but two of them are pre-installed with Python, so really you only need to worry about packaging the five above.
-
+## Running Locally
+1. Clone repo and `cd source-reminder`
+2. Configure a local `docker-env` file with the needed environment variables
+    ```sh
+    cp docker-env-example docker-env
+    ```
+3. Run Source Reminder Lambda container
+    ```sh
+    # builds a local image "source-reminder/lambda:local"
+    # then runs a container from the image named "source-reminder" on localhost:9000
+    make run-container
+    ```
+4. From a new terminal window, invoke the lambda function (simulates a webhook request to execute Lambda)
+    ```sh
+    make test-container
+    # runs: curl -XPOST "http://localhost:9000/2015-03-31/functions/functioninvocations" -d '{}'
+    # In case of source-reminder, we send an empty data body in the request to invoke the function ("-d '{}'"), but for other lambda functions, like hoedown-helper, this data body can be configured to test different invokation scenarios.
+    ```
+5. The function's output will show up in the terminal running the `source-reminder` Lambda container (matches what would show up in AWS Cloudwatch logs on function invokation).
+6. _Optional_:  if `DRY_RUN=True`, no Slack messages will be sent.  The console will output what _would_ have been sent.  If you want to run the function locally and have it send the messages (running as production), then set `DRY_RUN=FALSE`.
```

Comment:
I added a few comments but then deleted them, sorry. Going to try again... I was a bit confused about where DRY_RUN was, I though it was a ENV variable and so added it to `docker_env` before running. Until I figured out it was in the `lambda_function.py` file. I don't know if it's worth adding where it's located or if its just me who didn't realize😅

Reply:
Thank you for flagging this @jbarrueta - definitely can see how that was unclear and confusing, I'd definitely expect/read that as environment variables, too.  

I had a smaller branch was going to convert the hard-coded values to configurable environment variables, but was worried about this PR being too big.  But it's a small commit so I added that here so now they _are_ environment variables.  Updated `README.md` and `docker-env-example` to reflect that change, and have those env vars configured in AWS Lambda, too (`DRY_RUN` and `MANUAL_RUN` both set to `False` in Lambda).  

`MANUAL_RUN` might be no longer needed now that we have a containerized way of running this locally, that would use the Lambda handler just like we do in production/Lambda, but for now figure it doesn't hurt to have that option there just in case.